CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ A comprehensive list of documents is found [here](docs/DocumentList.md).
 * To test new code, see [here](docs/Testing.md) for instructions.
 * A guide on how to do constant propagation for ONNX operations is found
   [here](docs/ConstPropagationPass.md).
-* A guide on how to analyze unknown dimensions and query their equality at compile time is found [here](docs/UnknownDimensionAnalysis).
+* A guide on how to analyze dynamic dimensions and query their equality at compile time is found [here](docs/DynamicDimensionAnalysis).
```

Comment:
+1 on standardizing the terms that we use. This greatly help our users long term. We should do more of that, thanks for fixing this here.